src/aap_eda/core/migrations/0042_alter_activation_webhooks.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.7 on 2024-08-01 18:59
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("core", "0041_alter_credentialtype_options_and_more"),
```

Comment:
there is a migration conflict from the previous merge, can you fix it?

Reply:
Fixed